content/docs/1.5.1/volumes-and-nodes/internal-data-structures.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Internal Data Structures
+weight: 3
+---
+
+**Table of Contents**
+- [Node Disk](#node-disk)
+  - [Filesystem-Type Disk](#filesystem-type-disk)
+  - [Block-Type Disk](#block-type-disk)
+- [Replica Directory on Node Disk](#replica-directory-on-node-disk)
+
+## Node Disk
+
+### Filesystem-Type Disk
+
+A filesystem-type disk contains a configuration file named `longhorn-disk.cfg` and a directory called `replicas` that stores replica directories for volumes. The `longhorn-disk.cfg` file contains a JSON object with a single key `diskUUID` whose value is a 36-character UUID string, as shown below:
+```
+{
+  "diskUUID": "${UUID}"
+}
+```
+
+### Block-Type Disk
+
+A block-type disk is managed by the Storage Performance Development Kit (SPDK). SPDK constructs a logical volume store (lvstore) that utilizes the super blob feature of the blobstore to hold the UUID and other metadata. The blobstore types are implemented within the blobstore itself and are persisted on disk.
```

Comment:
Is there any upstream doc we can reference the link here?

Reply:
Agreed.  I was thinking maybe https://spdk.io/doc/